From 1e66c5262bb46f325a2778849c1faef27ff915d3 Mon Sep 17 00:00:00 2001
From: Jakub Wilk <jwilk@jwilk.net>
Date: Fri, 22 Feb 2019 15:16:48 +0100
Subject: [PATCH] genmanpage.pl: Fix whitespace stripping regexp

Fixes weird spacing in devscripts(1) man page.

Before:

    check whether a /bin/sh script contains any common   bash-specific constructs.

After:

    check whether a /bin/sh script contains any common bash-specific constructs.

Signed-off-by: Jakub Wilk <jwilk@jwilk.net>
---
 doc/genmanpage.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/genmanpage.pl b/doc/genmanpage.pl
index 4fe0cf80..9d33c155 100644
--- a/doc/genmanpage.pl
+++ b/doc/genmanpage.pl
@@ -24,7 +24,7 @@ while (<>) {
         $package     = $1;
         $description = $2;
     } else {
-        s/^.{2}}//;
+        s/^  //;
         $description .= $_;
     }
 }
-- 
2.20.1

